### PR TITLE
SAK-33883 Provide option to export the display ID.

### PR DESCRIPTION
**Pull request by buckett**
For deployments that use a display ID it’s often more useful to have this included in the gradebook export. We don’t use this in-place of the EID as it’s not guaranteed to be round trip-able whereas the EID always is.

### CODE REVIEW

**Comment by ottenhoff** [comment]
This makes sense to me.

**Comment by ottenhoff** [comment]
Looks like this can't merge anymore. @bjones86 do you have any thoughts on this PR?

File: gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/importExport/ExportPanel.java [hunk 1]
```diff
@@ -250,6 +261,12 @@ private File buildFile(final boolean isCustomExport) {
 			if (!isCustomExport || this.includeStudentId) {
 				header.add(getString("importExport.export.csv.headers.studentId"));
 			}
+
+			if (isCustomExport && this.includeStudentDisplayId)
+			{
```
Comment (bjones86) [inline gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/importExport/ExportPanel.java]: Brace style doesn't match up here

**Comment by buckett** [comment]
@bjones86 Good spot, fixed and rebased.

**Comment by ottenhoff** [comment]
@bjones86 can you dismiss your previous review if this looks okay?

**Pull request merged**